pkg/ddc/base/runtime_helper.go

```diff
@@ -196,3 +190,31 @@ func (info *RuntimeInfo) transformTemplateWithCacheDirDisabled(template *common.
 	template.FuseContainer.VolumeMounts = utils.TrimVolumeMounts(template.FuseContainer.VolumeMounts, cacheDirNames)
 	template.VolumesToAdd = utils.TrimVolumes(template.VolumesToAdd, cacheDirNames)
 }
+
+func (info *RuntimeInfo) applyFuseSidecarPostStartScriptConfigMap(cmToApply *corev1.ConfigMap) error {
+	cm, err := kubeclient.GetConfigmapByName(info.client, cmToApply.Name, cmToApply.Namespace)
+	if err != nil {
+		return err
+	}
+
+	if cm == nil {
+		// ConfigMap not found, so create it
+		//todo log not found configMap
+		err = info.client.Create(context.TODO(), cmToApply)
+		if err != nil {
+			return nil
+		}
+	} else {
+		// ConfigMap found, check if updated
+		if !reflect.DeepEqual(cm.Data, cmToApply.Data) {
```

Comment:
I have concern that if update cm in concurrent way, and some of them are failed. How to handle it?

Reply:
Updated